en/components/general/wpf_to_angular_guide/one_way_binding.md

```diff
@@ -0,0 +1,118 @@
+  text = 'IgniteUI for Angular';
+
+  constructor() { }
+  ngOnInit() {}
+}
+```
+```html
+<h2>{{ text }}</h2>
+```
+
+### Interpolation
+
+In the code from above, we simply display some text in the HTML by using a binding to the value of the `text` property. In this case, we are using `interpolation` to create a one-way binding. We do this by typing double curly braces, the name of the property - in our case `text`, and two closing curly braces. Another way to achieve the same result is to create h2 tag and bind the `text` property to its innerHTML property, by using the `interpolation` syntax again:
+```html
+<h2 innerHTML="{{ text }}"></h2>
+```
+There are two important things about `interpolation`. First, everything inside the curly braces is rendered as a `string`.
+Second, everything inside the curly braces is referred to as a `template expression`. This allows us to do more complex things, such as concatenation. For example, let's concatenate some text with the value of the `text` property:
+```html
+<h2>{{"Welcome to " + text }}</h2>
+```
+The use of template expressions allows us to bind to javascript properties and methods as well. For example, we can bind to the `text` property's length which will result in the number 20:
+```html
+<h2>{{ text.length }}</h2>
+```
+We can also bind to methods of that property, for example to `toUpperCase()`:
+```html
+<h2>{{ text.toUpperCase() }}</h2>
+```
+This is a lot more powerful than the data binding in WPF and a lot easier to use too. We can even make mathematical calculations inside the template expression. For example, we can simply put **2 + 2** into the expression, and it will display the result, which is equal to 4:
+```html
+<h2>{{ 2 + 2 }}</h2>
+```
+One more thing that we can do is to bind to actual methods from the typescript file. Here is a short example on how to achieve this:
+```html
+<h2>{{ getTitle() }}</h2>
+```
+This `getTitle()` is a method defined in the typescript file. The result on the page is the returned value of that method:
+```typescript
+getTitle() {
+  return 'Simple Title';
+}
+```
+
+Although the `interpolation` looks quite powerful, it has its limitations, for example - it only represents a **string**.
+So let's create a simple boolean property in the component class:
+```typescript
+export class SampleComponent implements OnInit {
+
+  text = 'IgniteUI for Angular';
+  isDisabled = false;
+  constructor() { }
+...
+```
+We will now create a simple `input` of type text and bind the `isDisabled` property to the input's `disabled` property:
+```html
+<input type="text" disabled="{{ isDisabled }}">
+```
+The expected result is that the `input` should be enabled, but it's disabled. This is because the `interpolation` returns a string, but the input's disabled property is of boolean type and it requires a boolean value.
+In order for this to work correctly, Angular provides `property binding`. It's syntax is a bit different.
```

Comment:
"It's syntax is a bit different." - should be "Its"

Reply:
Addressed